docs/reference/scaling-policies.md

```diff
@@ -88,32 +88,53 @@ properties:
 type: object
 ```
 
-The following table defines what each of these fields means.
+The following tables define what each of these fields means.
+
+## Responsive Policy Specs 
+
+Scope: `spec`
+
+| Full Field Name           | Description                                                                                                                                                                    |
```

Comment:
I think the word `Full` here and especially later is confusing if we're taking out the common prefixes

Reply:
I guess the idea was to point out that we're listing the entire thing in the first two tables, but only the suffix with the common prefix/scope removed in the 3rd table. 

But maybe that's too confusing. I think I'll just remove the common prefix/scope from the 2nd table as well to keep the formatting consistent